src/components/molecules/search-results-list/search-results-list.jsx

```diff
@@ -43,11 +44,17 @@ const SearchResultsList = ({
 		);
 	});
 
+	const renderHelmet = () => {
+		return (
```

Comment:
why is this being handled by the search results list?  This is something that should be handled by the view.

Reply:
Will switch.

Reply:
Done!